academy/2-main-concepts/modules.md

```diff
@@ -271,24 +271,24 @@ x/{module_name}
 * `simulation/`: the module's simulation package defines functions used by the blockchain simulator application (`simapp`).
 * `spec/`: the module's specification documents outlining important concepts, state storage structure, and message and event type definitions.
 * The root directory includes type definitions for messages, events, and genesis state including the type definitions generated by Protocol Buffers.
-    * `abci.go`: the module's `BeginBlocker` and `EndBlocker` implementations. This file is only required if `BeginBlocker` and/or `EndBlocker` need to be defined.
-    * `codec.go`: the module's registry methods for interface types.
-    * `errors.go`: the module's sentinel errors.
-    * `events.go`: the module's event types and constructors.
-    * `expected_keepers.go`: the module's expected other keeper interfaces.
-    * `genesis.go`: the module's genesis state methods and helper functions.
-    * `keys.go`: the module's store keys and associated helper functions.
-    * `msgs.go`: the module's message type definitions and associated methods.
-    * `params.go`: the module's parameter type definitions and associated methods.
-    * `*.pb.go`: the module's type definitions generated by Protocol Buffers as defined in the respective `*.proto` files above.
+  * `abci.go`: the module's `BeginBlocker` and `EndBlocker` implementations. This file is only required if `BeginBlocker` and/or `EndBlocker` need to be defined.
+  * `codec.go`: the module's registry methods for interface types.
+  * `errors.go`: the module's sentinel errors.
+  * `events.go`: the module's event types and constructors.
+  * `expected_keepers.go`: the module's expected other keeper interfaces.
+  * `genesis.go`: the module's genesis state methods and helper functions.
+  * `keys.go`: the module's store keys and associated helper functions.
+  * `msgs.go`: the module's message type definitions and associated methods.
+  * `params.go`: the module's parameter type definitions and associated methods.
+  * `*.pb.go`: the module's type definitions generated by Protocol Buffers as defined in the respective `*.proto` files above.
```

Comment:
To me 2 spaces is less easy to see than 4. It makes me wonder whether it was intentional or a fat finger error. Personal preference...

Reply:
Yeah, I use [`markdownlint`](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint) to lint markdown on save. By default, it's 2 spaces, but I will make it here to 4 spaces ➡️ I've copied the same `.markdownlint.json` we use in the sdk: https://github.com/cosmos/cosmos-sdk/blob/main/.markdownlint.json